pkg/balancer-js/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@balancer-labs/balancer-js",
```

Comment:
Is the name final?

Reply:
It feels like a fairly standard choice. I've asked in #integrations-internal if anyone has strong feelings but unless anyone objects then I think we can say it's final.